components/search_results_item/__snapshots__/search_results_item.test.tsx.snap

```diff
@@ -1231,7 +1456,7 @@ exports[`components/SearchResultsItem should match snapshot for deleted message
           >
             <Connect(PostTime)
               eventTime={1502715365009}
-              isPermalink={true}
+              isPermalink={false}
```

Comment:
Snapshot was wrong here because of bad casing of `deleted`, the test `components/search_results_item/search_results_item.tsx:194` was giving the wrong result.